hadoop-tools/hadoop-azure/src/main/java/org/apache/hadoop/fs/azurebfs/AzureBlobFileSystemStore.java

```diff
@@ -651,11 +651,20 @@ BlobCopyProgress handleCopyInProgress(final Path dstPath,
    * @throws AzureBlobFileSystemException exception thrown from
    * {@link AbfsClient#getBlobProperty(Path, TracingContext)} call
    */
-  BlobProperty getBlobProperty(Path blobPath,
-      TracingContext tracingContext) throws AzureBlobFileSystemException {
-    AbfsRestOperation op = client.getBlobProperty(blobPath, tracingContext);
+  BlobProperty getBlobProperty(Path blobPath, TracingContext tracingContext)
```

Comment:
if we need to change this method, lets add UT for getBlobProperty on root and some internal path.